include/Gaffer/SplinePlug.h

```diff
@@ -72,10 +72,20 @@ class SplinePlug : public ValuePlug
 		IECORE_RUNTIMETYPED_DECLARETEMPLATE( SplinePlug<T>, ValuePlug );
 		IE_CORE_DECLARERUNTIMETYPEDDESCRIPTION( SplinePlug<T> );
 
+		// TODO - this is where John had suggested this go in his initial email, but does it
+		// really make for this enum to be part of each separate templated version of SplinePlug?
+		enum Interpolation
```

Comment:
John, what do you think of this Interpolation going here?  Do you think it's an advantage to have many different versions of the Interpolation enum floating around?  I guess it makes the namespacing easier.